schedule/yam/agama_packages_unattended_ppc64le.yaml

```diff
@@ -9,3 +9,4 @@ schedule:
   - installation/grub_test
   - installation/first_boot
   - yam/validate/validate_packages
+  - yam/validate/validate_post_partitioning
```

Comment:
could we rename these files `agama_packages_*` to something more meaninful, like `extended` if I get it right they are just specific version for those architectures.